app/lib/efile/md/md502_calculator.rb

```diff
@@ -176,8 +178,6 @@ def calculate_exemption_amount
                             [175_001..200_000, 800],
                             [200_001..Float::INFINITY, 0]
                           ]
-                        else
-                          [[-Float::INFINITY..Float::INFINITY, 0]]
```

Comment:
all other filing statuses are handled here except for :dependent

I could leave the `else` statement here, since maybe that's safer?